.circleci/config.yml

```diff
@@ -182,11 +182,13 @@ jobs:
       - run:
           name: Testing
           command: |
-                INTEGRATION=1 go test -v -race `go list ./contrib/... | grep -v -e grpc.v12 -e google.golang.org/api`
+                INTEGRATION=1 go test -v -race `go list ./contrib/... | grep -v -e grpc.v12 -e google.golang.org/api -e cloud.google.com/go/pubsub`
                 go get google.golang.org/grpc@v1.29.0 # https://github.com/grpc/grpc-go/issues/3726
                 go test -v ./contrib/google.golang.org/api/...
                 go get google.golang.org/grpc@v1.2.0
                 go test -v ./contrib/google.golang.org/grpc.v12/...
+                go get cloud.google.com/go/pubsub@v1.6.1
```

Comment:
This "hack" is only needed for integrations which have imports that are not _the latest_ or for integrations that we have multiple versions for.

It should all work fine as long as what you have locally is the same as the result of doing `go get cloud.google.com/go/pubsub`

Reply:
P.S. Note that there was no entry in `go.mod` for this package, so applying https://github.com/DataDog/dd-trace-go/pull/721#discussion_r484432786 might just fix that and have CI work too.